### PR TITLE
[ERE-1824] Synchronise carer email when a user updates their email

### PR DESCRIPTION
**Pull request by ppettitau**

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 cachetools==4.1.1
-django-cache-memoize==0.1.10
```
Comment (ScriptSmith) [inline requirements/requirements.txt:2]: This is still needed for MIMS

File: mnd/mnd/models.py [hunk 2]
```diff
@@ -96,6 +98,15 @@ def clean(self):
             self.email = self.email.lower()
 
 
+@receiver(user_email_updated)
+def update_carer_email(sender, user, **kwargs):
```
Comment (ScriptSmith) [inline mnd/mnd/models.py:102]: When I updated the carers email, it appeared to go successfully, and the carer still appears to be able to see the patient. However, on the principal caregiver management page, it says that the carer hasn't been registered:

image

**Review by ScriptSmith — CHANGES_REQUESTED** [review]

File: requirements/requirements.txt (hunk 1 above)
Comment (ppettitau) [reply requirements/requirements.txt:2]: It's in base rdrf/requirements.txt now, so I thought I could remove it? Or is it still necessary to be at this level?

File: mnd/mnd/models.py (hunk 2 above)
Comment (ppettitau) [reply mnd/mnd/models.py:102]: Good pick up. Looks like there was yet another email field that needed to be updated 😬

File: requirements/requirements.txt (hunk 1 above)
Comment (ScriptSmith) [reply requirements/requirements.txt:2]: Oh, true :+1:

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**